homeassistant/components/asuswrt/config_flow.py

```diff
@@ -10,7 +10,7 @@
 from asusrouter import AsusRouterError
 import voluptuous as vol
 
-from homeassistant.components.device_tracker import (
+from homeassistant.components.device_tracker.const import (
```

Comment:
mmm this is not right

We should never import from other component sub-modules - and this should be blocked by pylint

Reply:
https://github.com/home-assistant/core/blob/afeaf2409fd20b04815615c566de714423199795/pylint/plugins/hass_imports.py#L329

Reply:
it can always be avoided! Created a new PR with an alternative fix for the problem